File: cartellediprogetto/Angular/CapstoneProjectFS/src/app/interfaces/filmtv.ts
export interface Filmtv {
  id:number,
  titolo:string,
  prezzo:string,
  durata:string,
  regista:string,
  attori:string,
  casaProduzione:string,
  formato:string,
  codiceControllo:string,
  quantita:string,
  immagine:string
}

export interface FilmTvCarrelloListaPreferiti {
  id:number,
  titolo:string,
  prezzo:string,
  durata:string,
  regista:string,
  attori:string,
  casaProduzione:string,
  formato:string,
  codiceControllo:string,
  quantita:string,
  immagine:string,
  nelCarrello:boolean,
  nellaLista:boolean,
  acquistabile:boolean
}

File: cartellediprogetto/CapstoneProjectFS/src/main/java/com/epicode/andreacursi/capstoneproject/entities/FilmTv.java
package com.epicode.andreacursi.capstoneproject.entities;

import javax.persistence.Entity;
import javax.persistence.EnumType;
import javax.persistence.Enumerated;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import com.epicode.andreacursi.capstoneproject.model.Formato;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Entity
public class FilmTv {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String titolo;
	private double prezzo;
	private int durata;
	private String regista;
	private String attori;
	private String casaProduzione;
	@Enumerated(EnumType.STRING)
	private Formato formato;
	private String codiceControllo;
	private int quantita;
	private String immagine;
	
}
